File: 268-missing-number/missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    nums.sort(function(a, b){return a - b});
    console.log(nums)
        let a = nums.length;
        for(let i = 0; i<nums.length; i++){
            if(i !== nums[i]){
                a = i;
                console.log(a)
                break;
            }
        }
        return a;
};

File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        Arrays.sort(nums);
        int a = nums.length;
        for(int i = 0; i<nums.length; i++){
            if(i != nums[i]){
                a = i;
                break;
            }
        }
        return a;
    }
}